sdk/src/main/java/org/eclipse/velocitas/sdk/grpc/GrpcCall.kt

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.eclipse.velocitas.sdk.grpc
+
+import io.grpc.stub.StreamObserver
+import org.eclipse.velocitas.sdk.logging.Logger
+
+abstract class GrpcCall {
+    // grpc::ClientContext context
+    var isComplete = false
+}
+
+/**
+ * A GRPC call where a request is followed up by a single reply.
+ *
+ * @param TRequestType The data type of the request.
+ * @param TResponseType The data type of the response.
+ */
+class SingleResponseGrpcCall<TRequestType, TResponseType>(
```

Comment:
These are just here b/c of gRPC's lacking async API in C++ - consider using something like https://github.com/rouzwawi/grpc-kotlin instead. Let's try to remove these classes.

Reply:
Added official implementation for https://github.com/grpc/grpc-kotlin. Henceforth the API of BrokerGrpcFacade also changed from ListenableFutures to suspend functions and Flows.